src/SFA.DAS.AssessorService.Web/Controllers/Apply/StandardController.cs

```diff
@@ -46,42 +49,59 @@ public async Task<IActionResult> Search(StandardViewModel model)
                 return RedirectToAction(nameof(Index), new { model.Id });
             }
 
-            var standards = await _apiClient.GetStandards();
+            var standards = await _standardVersionApiClient.GetAllStandardVersions();
```

Comment:
I would be tempted to add a GetLatestStandardVersions on the StandardVersionController to extend it. What do you think?

Reply:
I assume you mean to move the grouping to SQL. Can we leave it for now and make the change in a future story, as I think there is definitely a requirement for this functionality?

Reply:
Added Sub task to SV-564 https://skillsfundingagency.atlassian.net/browse/SV-822